shuup/front/apps/personal_order_history/templates/shuup/personal_order_history/order_detail.jinja

```diff
@@ -15,7 +15,8 @@
 {% endblock %}
 
 {% block dashboard_content %}
-    <div class="row">
+    <h1 class="page-header">Order #{{ order.id }}{{ render_action_buttons(order) }}</h1>
+    <div class="row order-dashboard-container">
```

Comment:
The added text should be translatable and use the order.identifier. There's already an example at line 4 on how to do that:
```
{% trans identifier = order.identifier %}Order {{ identifier }}{% endtrans %}
```
Is the h1 element really needed here, since there already is a h1 title rendered by the content_title macro (line 6)?

Would it look bad if those actions were placed there without the title? (Maybe to some other place?)